nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateCsv.java

```diff
@@ -120,11 +120,20 @@ public class ValidateCsv extends AbstractProcessor {
             .description("The schema to be used for validation. Is expected a comma-delimited string representing the cell "
                     + "processors to apply. The following cell processors are allowed in the schema definition: "
                     + allowedOperators.toString() + ". Note: cell processors cannot be nested except with Optional.")
-            .required(true)
+            .required(false)
```

Comment:
This should have a `dependsOn(HEADER, "false")`, unless you can provide the explicit schema yet there be a header line that should be ignored.

Reply:
I think the "unless" part is true here. But now there are merge conflicts, sorry I lost track of this

Reply:
I've updated the documentation.